test/e2e/suite/issuers/vault/certificate/approle.go

```diff
@@ -18,6 +18,7 @@ package certificate
 
 import (
 	"context"
+	"github.com/jetstack/cert-manager/test/e2e/framework/helper/featureset"
```

Comment:
this import is in the wrong place; see our new [Coding Conventions](https://cert-manager.io/docs/contributing/coding-conventions/) for a description

Reply:
I will fix this, of course. Would it make sense to add some kind of script that updates/verifies your coding conventions? As `./hack/update-gofmt.sh` and `./hack/verify-gofmt.sh`? Some kind of linting early in the CI build could also be nice.

Reply:
Yeah that's on the roadmap alongside a better release process - this stuff _should_ be automated.

I think most of us in the team have editors which run `goimports` on save, so for a large amount of changes to cert-manager that's like a "pre-commit" CI step (but again, should be automated!)

Reply:
Aha, so I just have to run goimports! 😄 Different open source projects tend to have a bit different coding conventions, so I usually turn everything off when working with PRs - to not introduce unnecessary changes.

Reply:
Fixed